src/lib/Components/Bidding/Components/Title.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { StyleSheet, TextProperties } from "react-native"
+
+import Serif from "../../Text/Serif"
+
+export class Title extends React.Component<TextProperties> {
+  render() {
+    const { children, style, ...props } = this.props
+
+    return (
+      <Serif style={[styles.default, style, styles.required]} {...props}>
+        {children}
+      </Serif>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Sorry if this wasn't clear on the other comment!

We're trying to move to Styled Components rather than using StyleSheets. Here's the [Inquiry screen](https://github.com/artsy/emission/blob/master/src/lib/Containers/Inquiry.tsx) as an example. It's a bit of a big file, but you can see towards the top of the file there are these different view components defined (e.g. [`TitleView`](https://github.com/artsy/emission/blob/master/src/lib/Containers/Inquiry.tsx#L40), [`Header`](https://github.com/artsy/emission/blob/master/src/lib/Containers/Inquiry.tsx#L28), and [`Content`](https://github.com/artsy/emission/blob/master/src/lib/Containers/Inquiry.tsx#L56)). These are then used in [the render method](https://github.com/artsy/emission/blob/master/src/lib/Containers/Inquiry.tsx#L200-L240). 

You can also see that there is a custom Style array being used, because there was an exception with the keyboard avoiding button view, that needed the styles passed through. But generally, that's left for special cases.